clicache/src/QueryService.hpp

```diff
@@ -51,80 +60,80 @@ namespace Apache
       /// <summary>
       /// Provides a query service.
       /// </summary>
-      generic<class TKey, class TResult>
       public ref class QueryService sealed
       {
       public:
 
         /// <summary>
         /// Get a <c>Query</c> object to enable querying.
         /// </summary>
-        //generic<class TResult>
+        generic<class TResult>
         Query<TResult>^ NewQuery( String^ query );
         /// @nativeclient
         /// <summary>
         /// Get a <c>CqQuery</c> object to enable continuous querying.
         /// </summary>
         /// @endnativeclient
-        //generic<class TKey, class TResult>
-        CqQuery<TKey, TResult>^ NewCq( String^ query, CqAttributes<TKey, TResult>^ cqAttr, bool isDurable );
+        generic<class TKey, class TResult>
+          CqQuery<TKey, TResult>^ NewCq(String^ query, CqAttributes<TKey, TResult>^ cqAttr, bool isDurable);
+  
         /// @nativeclient
         /// <summary>
         /// Get a <c>CqQuery</c> object to enable continuous querying.
         /// </summary>
         /// @endnativeclient
-        //generic<class TKey, class TResult>
+        generic<class TKey, class TResult>
         CqQuery<TKey, TResult>^ NewCq( String^ name, String^ query, CqAttributes<TKey, TResult>^ cqAttr, bool isDurable );
         /// @nativeclient
         /// <summary>
         /// Close all  <c>CqQuery</c> on this client.
         /// </summary>
         /// @endnativeclient
-	void CloseCqs();
+	      void CloseCqs();
```

Comment:
Nit - spacing seems a little off.  Mixed tabs/spaces. Same as some of other methods below.

Reply:
Went through and fixed the spacing...